karma.conf.js

```diff
@@ -18,43 +19,23 @@ const browserPlatformPairs = {
 module.exports = config => {
   let bundleDirpath;
   const cfg = {
-    frameworks: ['browserify', 'mocha'],
+    frameworks: ['rollup', 'mocha'],
     files: [
       // we use the BDD interface for all of the tests that
       // aren't interface-specific.
       'test/unit/*.spec.js'
     ],
-    preprocessors: {
-      'test/**/*.js': ['browserify']
-    },
-    browserify: {
-      debug: true,
-      configure: function configure(b) {
-        b.ignore('chokidar')
-          .ignore('fs')
-          .ignore('glob')
-          .ignore('path')
-          .ignore('supports-color')
-          .ignore('./lib/esm-utils.js')
-          .ignore('./lib/cli/*.js')
-          .ignore('./lib/nodejs/serializer.js')
-          .ignore('./lib/nodejs/worker.js')
-          .ignore('./lib/nodejs/buffered-worker-pool.js')
-          .ignore('./lib/nodejs/parallel-buffered-runner.js')
-          .ignore('./lib/nodejs/reporters/parallel-buffered.js')
-          .on('bundled', (err, content) => {
-            if (err) {
-              throw err;
-            }
-            if (bundleDirpath) {
-              // write bundle to directory for debugging
-              fs.writeFileSync(
-                path.join(bundleDirpath, `mocha.${Date.now()}.js`),
-                content
-              );
-            }
-          });
-      }
+    plugins: [
+      'karma-mocha',
+      'karma-mocha-reporter',
+      'karma-sauce-launcher',
+      'karma-chrome-launcher',
+      rollupPlugin
+    ],
+    rollup: {
+      configFile: 'rollup.config.js',
+      include: ['test/**'],
+      bundlePath: bundleDirpath
     },
     reporters: ['mocha'],
     colors: true,
```

Comment:
`client.mocha.opts`: the `opts` option has been removed in Mocha v8.0.0.

Reply:
Fixed in 6be6606c